From fcd66e9f32107ce62b96b6e6e0b7565adda99710 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Emilio=20L=C3=B3pez?= <turl@tuxfamily.org>
Date: Sun, 13 May 2012 00:08:50 +0000
Subject: [PATCH 428/944] arm: sun4i: pm: fix build

There were some type redefinition errors when building, fix them by using the right
header and not defining them locally.
---
 arch/arm/mach-sun4i/pm/standby/common.h | 13 +++----------
 1 file changed, 3 insertions(+), 10 deletions(-)

diff --git a/arch/arm/mach-sun4i/pm/standby/common.h b/arch/arm/mach-sun4i/pm/standby/common.h
index ede1e28..30c5ce9 100644
--- a/arch/arm/mach-sun4i/pm/standby/common.h
+++ b/arch/arm/mach-sun4i/pm/standby/common.h
@@ -15,18 +15,11 @@
 * Update  : date                auther      ver     notes
 *********************************************************************************************************
 */
-#ifndef __COMMON_H__
-#define __COMMON_H__
 
-typedef signed char         __s8;
-typedef unsigned char       __u8;
-typedef signed short        __s16;
-typedef unsigned short      __u16;
-typedef signed int          __s32;
-typedef unsigned int        __u32;
-typedef signed long long    __s64;
-typedef unsigned long long  __u64;
+#include <linux/types.h>
 
+#ifndef __COMMON_H__
+#define __COMMON_H__
 
 static inline __u64 standby_uldiv(__u64 dividend, __u32 divisior)
 {
-- 
1.8.0

